openapi/index.yaml

```diff
@@ -66,3 +79,149 @@ definitions:
       - statusQueryGetUri
       - sendEventPostUri
       - terminatePostUri
+  ProblemJson:
+    $ref: "https://raw.githubusercontent.com/pagopa/io-functions-commons/v10.2.3/openapi/definitions.yaml#/ProblemJson"
+  EligibilityCheckStatus:
+    type: string
+    description: |
+      Eligibility check status.
+      - USER_NOT_FOUND    User not found in INPS database
+      - ISEE_NOT_FOUND    User found but there's no ISEE data
+      - ELIGIBILE         The user is eligible for the bonus
+      - INELIGIBLE        The user is not eligible for the bonus
+    x-extensible-enum:
+      - USER_NOT_FOUND
+      - ISEE_NOT_FOUND
+      - ELIGIBILE
+      - INELIGIBLE
+  FamilyMember:
+    type: object
+    properties:
+      fiscal_code:
+        $ref: '#/definitions/FiscalCode'
+      name:
+        type: string
+        minLength: 1
+      surname:
+        type: string
+        minLength: 1
+    required:
+      - name
+      - surname
+  EligibilityCheck:
+    type: object
+    title: Eligibility check
+    description: Eligibility check
+    properties:
+      id:
+        type: string
+        minLength: 1
+      status:
+        $ref: "#/definitions/EligibilityCheckStatus"
+      family_members:
+        type: array
+        items:
+          $ref: "#/definitions/FamilyMember"
+      max_amount:
+        description: Max bonus amount in euro cents
+        type: integer
+        minimum: 0
+        maximum: 50000
+      max_tax_benefit:
+        description: Max bonus tax benefit in euro cents
+        type: integer
+        minimum: 0
+        maximum: 50000
+    required:
+      - id
+      - status
+      - max_amount
+      - max_tax_benefit
+      - family_members
+  QrCode:
```

Comment:
QrCode is generated backend side

Reply:
Ok removed. Actually I left some definitions not used yet, but I guess they'll be needed in future operations. Please review the definitions when you'll add a new operation